test/gridengine/setup_tool.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cp tool_conf.xml config
+cp galaxy.ini /etc/galaxy/galaxy.ini
```

Comment:
I check `latest` and `dev` container.
Their `GALAXY_CONFIG_JOB_WORKING_DIRECTORY` is setup what I want to write in galaxy.ini.